components/inputs/input-text.js

```diff
@@ -140,8 +141,9 @@ class InputText extends FormElementMixin(RtlMixin(LitElement)) {
 				:host([hidden]) {
 					display: none;
 				}
-				label {
-					display: block;
+				.d2l-input-label {
+					display: inline-block;
+					vertical-align: bottom;
```

Comment:
Block elements aren't great for skeletizing, since the skeleton fills the whole width instead of just the width of the text. Instead of making all `d2l-input-label`s inline -- which would break all the things -- I'm going to be a bit more surgical.

Reply:
One side effect of this block -> inline change is that text inputs with labels have shifted down by `1px`, making the visual diffs `1px` taller. This mostly impacted the date input visual diffs.